packages/phone-number-privacy/combiner/test/end-to-end/get-contact-matches.test.ts

```diff
@@ -1,7 +1,7 @@
-import { OdisUtils } from '@celo/identity/lib/odis'
-import { ErrorMessages } from '@celo/identity/lib/odis/query'
 import { ensureLeading0x } from '@celo/utils/lib/address'
 import 'isomorphic-fetch'
+import { OdisUtils } from '../../../../sdk/identity/lib/odis'
+import { ErrorMessages } from '../../../../sdk/identity/lib/odis/query'
```

Comment:
Why this is change needed? Intuitively, this is more fragile since this code is now relying the packages being in relatively located in the monorepo structure.